spring-cloud-gcp-samples/spring-cloud-gcp-logging-sample/src/main/appengine/app.yaml

```diff
@@ -0,0 +1,9 @@
+runtime: java
```

Comment:
I'd erase this file and not use App Engine in our sample.

Reply:
I think there's some value to get people to try this out on GAE. It could introduce them to using GCP and also some other examples for other GCP products require you to setup a GCP account anyway, and it would be good to use this here. 

I mean really i think it's much more likely somebody is using stackdriver GCP logging if they're also using GCP to host their application...